### PR TITLE
Feature/add share and like function

### PR DESCRIPTION
**Pull request by hstaatsxx**
I added SingleFavoritePage to display each favorite page
and updated FavoritePages similar to ArchivePage.

Now the FavoritePage shows a list of favorite pages just like the archive page does.
Removing and adding a single article works.

I solved the connection/loading method from the favorite page to every single favorite page with the issue and article Id.

`<Link to={`/favorites/${favorite.issue}/${favorite.articleId}`} key={idx}>`
 
and added under Routes 

`<Route exact path="/favorites/:issue/:articleId" component={SingleFavoriePage}/>`

If this concerns any exposure of internal Ids to outside please comment. I will consider another way.

### CODE REVIEW

**Comment by DevJacobi** [comment]
Update: we postpone the PR until we have clarified with Oli and Jonathan how like and share should finally look like.

**Comment by hstaatsxx** [comment]
@constructive-news any updates regarding this issue?

**Comment by constructive-news** [comment]
Hi, I onboarded a new UX / UI designer today and she will work out a solution for that. Will take at least a few more days though. But we’re on it.

Jonathan

**Comment by constructive-news** [comment]
Thanks, Hyunjin!

And yes, we’ll definitely keep that in mind.

Jonathan

File: src/pages/SingleFavoritePage.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+const { Browser, Share } = Plugins;
+
+interface Props extends RouteComponentProps<{
+    issue: string,
+    articleId: string
+}> { }
+
+
+export const SingleFavoriePage: React.FC<Props> = ({ match }) => {
```
Comment (DavidKeanu) [inline src/pages/SingleFavoritePage.tsx:29]: Is the naming **SingleFavoriePage** ok like that?

**Comment by DavidKeanu** [comment]
Testing Video: 
https://user-images.githubusercontent.com/58940306/211207903-e7429ff9-4032-4ecc-be4b-07ff04d70ed0.mp4

**Pull request closed without merging**